src/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit/Models/Propagated.cpp

```diff
@@ -0,0 +1,644 @@
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+/// @project        Open Space Toolkit ▸ Astrodynamics
+/// @file           OpenSpaceToolkit/Astrodynamics/Trajectory.cpp
+/// @author         Antoine Paletta <antoine.paletta@loftorbital.com>
+/// @license        Apache License 2.0
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+#include <OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit/Models/Propagated.hpp>
+#include <OpenSpaceToolkit/Physics/Environment/Objects/CelestialBodies/Earth.hpp>
+
+#include <OpenSpaceToolkit/Core/Error.hpp>
```

Comment:
I don't think we need this defined do we?

Reply:
Depends if this is defined in a higher level `.hpp` file imported here (and if you're using it here ofc)